kubernetes/INSTALL.md

```diff
@@ -7,11 +7,11 @@
     
     kubectl config set-context `kubectl config view | grep current-context | awk '{print $2}'` --namespace <ns>
 
-**Install Istio infra**
+**Install Istio control plane and management plane**
 
```

Comment:
Especially for those coming from networking background, the separation is important.

Shall we go back to infra?